base/libgit2/consts.jl

```diff
@@ -263,4 +263,12 @@ module Consts
                         CREDTYPE_SSH_INTERACTIVE    = Cuint(1 << 4),
                         CREDTYPE_USERNAME           = Cuint(1 << 5),
                         CREDTYPE_SSH_MEMORY         = Cuint(1 << 6))
+
+    @enum(GIT_CONFIG, CONFIG_LEVEL_DEFAULT = 0,
```

Comment:
I'll do it, but I do not feel like putting every enum value into docs. Enum values usually are self-explanatory.